Makefile

```diff
@@ -161,6 +161,8 @@ package/Dockerfile.submariner-operator: bin/submariner-operator
 
 package/Dockerfile.submariner-operator-index: packagemanifests
 
+package/Dockerfile.submariner-diagnose: bin/subctl
+
```

Comment:
This shouldn’t be necessary, `docker_deps` in Shipyard’s `Makefile.images` extracts `COPY` statements to calculate prerequisites.

Reply:
We should probably fix this on Shipyard then, have you opened an issue there?

Reply:
https://github.com/submariner-io/shipyard/issues/727